internal/pkg/deploy/cloudformation/stack/validate.go

```diff
@@ -411,7 +420,7 @@ func validateContainerPath(input string) error {
 // SNS Topic intended for a publisher.
 func validatePubSubName(name *string) error {
 	if name == nil || len(aws.StringValue(name)) == 0 {
-		return errMissingPublishTopicField
+		return errMissingPubSubTopicField
```

Comment:
🤔  This is not part of this particular PR but I'm wondering why we need to pass `*string` to this method? Looks like we should be fine passing just `string` and it would save us several pointer <-> non-pointer conversions.